ci_support/run_docker_build.sh

```diff
@@ -58,7 +58,9 @@ conda clean --lock
 conda install --yes --quiet conda-forge-build-setup
 source run_conda_forge_build_setup
 
+conda install --yes conda-build=3
```

Comment:
@jakirkham @scopatz 
It looks like using conda-build=3 fixes the post build error we were having.

Is that an okay temporary solution to get the package updated?

Reply:
Yes I think that is fine. We are moving to conda-build=3 anyway